src/System.Management.Automation/engine/remoting/client/Job.cs

```diff
@@ -3907,6 +3907,15 @@ public override DebuggerCommandResults ProcessCommand(PSCommand command, PSDataC
             return _wrappedDebugger.ProcessCommand(command, output);
         }
 
+        /// <summary>
+        /// Adds the provided set of breakpoints to the debugger.
+        /// </summary>
+        /// <param name="breakpoints">Breakpoints.</param>
+        public override void SetBreakpoints(IEnumerable<Breakpoint> breakpoints)
+        {
+            _wrappedDebugger?.SetBreakpoints(breakpoints);
```

Comment:
Null check is not really needed since _wrappedDebugger is assigned in constructor.